content/geometry/object-environment/environment-structure.adoc

```diff
@@ -0,0 +1,482 @@
+= Model structure
+
+== Structure overview
+
+[plantuml]
+----
+legend
+[[../geometry/object-classes.html#_environment_t Environment_Root (T)]]
+|_[[../geometry/object-classes.html#_terrain_t Terrain]]
+|_[[../geometry/object-classes.html#_environment_objects_t Objects]]
+  |_ [[../geometry/object-classes.html#_building_idx Building_<building_idx>]]
+  |_ [[../geometry/object-classes.html#_vegetation_idx Vegetation_<vegetation_idx>]]
+  |_ [[../geometry/object-classes.html#_other_idx Other_Objects_<object_idx>]]
```

Comment:
Proposal: Remove the indexes here and just make Buildings, Vegetation and Other group nodes, where any number of meshes can be in.

Reply:
- [ ] Has been changed: removed indices in objects' subclasses.